libsolidity/formal/BMC.cpp

```diff
@@ -1192,6 +1194,15 @@ BMC::checkSatisfiableAndGenerateModel(vector<smtutil::Expression> const& _expres
 	vector<string> values;
 	try
 	{
+		if (m_settings.printQuery)
+		{
+			auto portfolio = dynamic_cast<smtutil::SMTPortfolio*>(m_interface.get());
+			string smtlibCode = portfolio->dumpQuery(_expressionsToEvaluate);
+			m_errorReporter.info(
+				6240_error,
+				"; BMC: Requested query\n" + smtlibCode
```

Comment:
```suggestion
				"BMC: Requested query:\n" + smtlibCode
```
Sorry for the back and forth, but I just noticed that this goes anyway in front of `Info: `, so let's remove the `;`.